src/screens/Bridge/Exchange-CrossChain/OfferCard.js

```diff
@@ -67,6 +67,7 @@ type Props = {
   onFetchSortingOfferInfo?: (offerInfo: ExchangeOffer) => void,
   isSelected?: ?boolean,
   onFeeInfo?: (feeInfo: ?TransactionFeeInfo) => void,
+  onEstimating?: (estimating: boolean) => void,
```

Comment:
I think the type of setLoading isn't the same as whats being used here

Reply:
So the thing is the function type you've define can take any function here  with a Boolean argument, but shouldn't we make it more typed i mean see the type of setLoading and use that here over the generic function type